metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_config.py

```diff
@@ -112,15 +112,22 @@ class ProfileConfig(GEProfilingConfig):
 
 
 class DremioSourceMapping(EnvConfigMixin, PlatformInstanceConfigMixin, ConfigModel):
-    platform: Optional[str] = Field(
-        default=None,
+    platform: str = Field(
         description="Source connection made by Dremio (e.g. S3, Snowflake)",
     )
-    source_name: Optional[str] = Field(
-        default=None,
+    source_name: str = Field(
         description="Alias of platform in Dremio connection",
     )
 
+    @root_validator
```

Comment:
We shouldn't need this validator now that both are marked as str. Pydantic would make sure of it.